docs/npm-lerna.md

```diff
@@ -0,0 +1,21 @@
+# Publishing to npm
+
+Shipit can be used to publish code to npm.
+If your project has a `package.json`, and doesn’t have any other deploy method set up, Shipit will publish it to npm.
+
+[Lerna](https://github.com/lerna/lerna) is a tool for managing projects with multiple packages.
```

Comment:
We might want to split lerna and npm into their own documents, or have lerna be at the end so it's clear it's it's own thing

Reply:
makes sense 👍 I put the Lerna info at the end.